src/compiler/types.ts

```diff
@@ -3676,6 +3676,8 @@ namespace ts {
 
     export type CompilerOptionsValue = string | number | boolean | (string | number)[] | string[] | MapLike<string[]> | PluginImport[] | null | undefined;
 
+    export type StrictOptionName = "noImplicitAny" | "noImplicitThis" | "strictNullChecks" | "strictFunctionTypes" | "alwaysStrict";
+

```

Comment:
Can this not be in core (since the function itself is internal why not type as well be internal)